editor/plugins/node_3d_editor_gizmos.cpp

```diff
@@ -244,6 +246,32 @@ void EditorNode3DGizmo::add_mesh(const Ref<Mesh> &p_mesh, const Ref<Material> &p
 	instances.push_back(ins);
 }
 
+void EditorNode3DGizmo::_update_bvh() {
+	ERR_FAIL_NULL(spatial_node);
+
+	Transform3D transform = spatial_node->get_global_transform();
+
+	float effective_icon_size = selectable_icon_size > 0.0f ? selectable_icon_size : 0.0f;
+	Vector3 icon_size_vector3 = Vector3(effective_icon_size, effective_icon_size, effective_icon_size);
+	AABB aabb(spatial_node->get_position() - icon_size_vector3, icon_size_vector3);
```

Comment:
Bounding box doesn't match the one defined [here](https://github.com/godotengine/godot/blame/e25f3c0d38d457b15a63720240736f564ce0501b/editor/plugins/node_3d_editor_gizmos.cpp#L348), will fix